CHANGES_UNRELEASED.md

```diff
@@ -18,3 +18,11 @@ Use the template below to make assigning a version number during the release cut
   - Description of the change with a link to the pull request ([#0000](https://github.com/mozilla/application-services/pull/0000))
 
 -->
+
+## Sync Manager
+
+### What's Changed
+  - Sync manager methods have now been standardized to support all available
+    engines.  Before each method supported its own subset of the engines
+    (`wipe()` and `reset()` didn't support tabs, `reset_all()` didn't support
+    tabs or logins, etc).
```

Comment:
Maybe I should have called this a breaking change, since the behavior of `reset_all()` is different.  But it seems more like a bugfix to me and also I don't think Fenix ever calls `resetAll()`.